.travis.yml

```diff
@@ -3,42 +3,49 @@ services: docker
 language: bash
 addons:
     apt:
-        config:
-            retries: true
         update: true
         packages:
-            - jq
             - rpm2cpio
             - cpio
 env:
-    global:
-        - VERSION=4.0.0-5
-        # See qemu-user-static's RPM spec file on Fedora to check the new version.
-        # https://src.fedoraproject.org/rpms/qemu/blob/master/f/qemu.spec
-        - DOCKER_SERVER=docker.io
-        # - DOCKER_SERVER=quay.io
-        # Git repository
-        - REPO=multiarch/qemu-user-static
-        # Container repository
-        - DOCKER_REPO=$DOCKER_SERVER/multiarch/qemu-user-static
-        # - DOCKER_REPO=$DOCKER_SERVER/your_username/qemu-user-static
-        - PACKAGE_URI="https://kojipkgs.fedoraproject.org/packages/qemu/4.0.0/5.fc31/x86_64/qemu-user-static-4.0.0-5.fc31.x86_64.rpm"
-        - PACKAGE_FILENAME=$(basename "$PACKAGE_URI")
+  global:
+    # See qemu-user-static's RPM spec file on Fedora to check the new version.
+    # https://src.fedoraproject.org/rpms/qemu/blob/master/f/qemu.spec
+    # Container repository
+    - PACKAGE_URI="https://kojipkgs.fedoraproject.org/packages/qemu/4.0.0/5.fc31/x86_64/qemu-user-static-4.0.0-5.fc31.x86_64.rpm"
+    - PACKAGE_FILENAME=$(basename "$PACKAGE_URI")
+
+stages:
+  - build
+  - test
+  - name: deploy
+    if: tag IS present
+
+
 before_script:
-    - wget --content-disposition $PACKAGE_URI
-    - rpm2cpio $PACKAGE_FILENAME | cpio -dimv
-script:
+    - wget --content-disposition ${PACKAGE_URI}
+    - rpm2cpio ${PACKAGE_FILENAME} | cpio -dimv
     - ./generate_tarballs.sh
-    - |
-      if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
-          ./publish.sh -v "$VERSION" -t "$GITHUB_TOKEN" -r "$REPO"
-      fi
-    - ./update.sh -v "$VERSION" -r "$REPO" -d "$DOCKER_REPO"
-    - docker images
-    - ./test.sh -d "$DOCKER_REPO"
-after_success:
-    - |
-      if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
-          docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD" "$DOCKER_SERVER" && \
-              docker push "$DOCKER_REPO"
-      fi
+
+jobs:
+  include:
+    - stage: build
+      name: "Build Docker image for amd64"
```

Comment:
Can you change "amd64" to "x86_64"? Because "x86_64" is already used in other parts.